src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -153,12 +153,13 @@ private void changeCheckInServicePointId(UUID servicePointId) {
     write(representation, "checkinServicePointId", servicePointId);
   }
 
-  private void changeItemStatusToDeclaredLost() {
+  public Loan changeItemStatusForItemAndLoan(ItemStatus itemStatus) {
```

Comment:
Why does there need to be two methods for changing the item status?

Reply:
One method changes item status in item representation, another method changes an item status in loan representation

Reply:
Hmm, there is a good chance that neither of these methods should exist on the loan, and they are symptoms of poor separation between aggregate and read models.